primes/bash/compute.bash

```diff
@@ -19,10 +19,14 @@ number=2
 echo begin
 while [[ $nb_primes -lt $PRIME_COUNT ]]; do
     if is_prime $number; then
-        echo $number
-        primes="$primes $number"
+        primes="$primes
+$number"
```

Comment:
Unfortunately, this adds an extra newline after the `begin`:

```
begin

2
3
end
```

Also, the `$number"` at the beginning of the line is ugly :p

By the way, why not use Bash arrays? I guess requiring Bash 4+ is OK.

Reply:
Fixed